ClientGUI/XNALinkButton.cs

```diff
@@ -47,7 +47,15 @@ public override void ParseAttributeFromINI(IniFile iniFile, string key, string v
 
         public override void OnLeftClick()
         {
+#if NET48
             Process.Start(URL);
+#else
+             _ = Process.Start(new ProcessStartInfo
+            {
+                FileName = URL,
+                UseShellExecute = true
+            });
```

Comment:
Why the `#if` condition? Can't you just use the .NET 6 code also on .NET 4.8?

Also why the `_` instead of just not saving the return value of `Process.Start`?

Reply:
Yes the added code also works for .NET 4.8. The only reason I left it is because I was unsure of how much you wanted to leave the original .NET 4.8 code intact. Perhaps you would've had concerns over changed behavior on .NET 4.8. I'll switch it to the new code.

_ is the discard symbol, so nothing is saved to a variable or memory or something like that. It makes it clear the developer did not accidently forget to use/inspect the return value of a function but intentionally chose to discard it.

Reply:
Code is updated